api/app/controllers/concerns/spree/api/v2/platform/nested_set_reposition_helper.rb

```diff
@@ -0,0 +1,22 @@
+module Spree
+  module Api
+    module V2
+      module Platform
+        module NestedSetRepositionHelper
```

Comment:
```suggestion
        module NestedSetRepositionConcern
```

as it's a Concern really not (rails) Helper